lean/components/config/project_config_manager.py

```diff
@@ -57,6 +57,23 @@ def get_local_id(self, project_directory: Path) -> int:
 
         return project_id
 
+    def get_latest_live_directory(self, project_directory: Path) -> Path:
+        """Returns the path of the latest live directory.
+
+        :param project_directory: the path to the project to retrieve the local id of
+        :return: the path of the latest live directory
+        """
+        live_root_dir = project_directory / "live"
+        if not live_root_dir.is_dir():
+            return None
+
+        live_dirs = []
+        for live_dir in sorted(list(live_root_dir.iterdir())):
+            if (live_dir / "log.txt").is_file():
+                live_dirs.append(live_dir)
+
+        return live_dirs[-1]
```

Comment:
I think should be able to sort the directories in such a way the first `live_dir` found is the one we want, right?